src/components/addonCollection/raffle/RaffleLocations.ts

```diff
@@ -1,45 +1,45 @@
 // P stands for position and number stands for the 3x3 grid from 0-2 on both axis
-export const P00 = {
+export const P00: object = {
```

Comment:
```suggestion
interface Position {
  justifyContent: string;
  alignItems: string;
}

export const P00: Position= {
```
Change for all objects